specs/core/1_custody-game.md

```diff
@@ -518,14 +517,14 @@ def process_bit_challenge(state: BeaconState,
     assert is_slashable_validator(challenger, get_current_epoch(state))
 
     # Verify the attestation
```

Comment:
worth assigning `attestation = challenge.attestation` to reduce line lengths throughout function

Reply:
Yeah, that shortens things a lot.